src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -181,11 +190,30 @@ def acr_connected_registry_update(cmd,  # pylint: disable=too-many-locals, too-m
 
     client_token_list = list(client_token_set) if client_token_set != current_client_token_set else None
 
+    # Add or remove from the current notifications list
+    add_notifications_set = set(list(add_notifications_list)) \
+        if add_notifications_list else set()
+
+    remove_notifications_set = set(list(remove_notifications_list)) \
+        if remove_notifications_list else set()
+
+    duplicate_notifications = set.intersection(add_notifications_set, remove_notifications_set)
+    if duplicate_notifications:
+        errors = sorted(duplicate_notifications)
+        raise CLIError(
```

Comment:
Could you please use the specific error type `ArgumentUsageError` instead of `CLIError` ?